tasks/Qc.wdl

```diff
@@ -396,6 +396,37 @@ task CollectDuplicateMetrics {
   }
 }
 
+# Build BAM/CRAM index
+task BuildBamIndex {
+  input {
+    File input_bam
+    File ref_dict
+    File ref_fasta
+    File ref_fasta_index
+    Int preemptible_tries
+  }
+
+  Float ref_size = size(ref_fasta, "GiB") + size(ref_fasta_index, "GiB") + size(ref_dict, "GiB")
+  Int disk_size = ceil(size(input_bam, "GiB") + ref_size) + 20
+
+  command {
+    java -Xms2000m -jar /usr/picard/picard.jar \
+      BuildBamIndex \
+      INPUT=~{input_bam}
+      ASSUME_SORTED=true \
+      REFERENCE_SEQUENCE=~{ref_fasta}
+  }
+  runtime {
+    docker: "us.gcr.io/broad-gotc-prod/picard-cloud:2.21.7"
+    memory: "3 GiB"
+    disks: "local-disk " + disk_size + " HDD"
+    preemptible: preemptible_tries
+  }
+  output {
+    File bam_index = glob("*.{b,cr}ai")
```

Comment:
A way to do this without `glob` might be something like:
```wdl
File bam_index = sub(sub(~{input_bam},"bam$","bai"),"cram$","crai")
```
(Not sure if these indexes are like `.bam.bai` or just `.bai`--will need tweaking if the former.)

Reply:
My impression is these are simply .bai or .crai files (no .bam or .cram):

From Picard docs: 
> Defaults to x.bai if INPUT is x.bam, otherwise INPUT.bai.

Reply:
If we're sure of that and feeling concise, we could probably get away with:
```wdl
File bam_index = sub(~{input_bam},"am$","ai")
```